server/src/main/java/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -1289,17 +1281,16 @@ public Network createGuestNetwork(CreateNetworkCmd cmd) throws InsufficientCapac
                 // if the guest network is for the VPC, if any External Provider are supported in VPC
                 // cidr will not be null as it is generated from the super cidr of vpc.
                 // if cidr is not null and network is not part of vpc then throw the exception
-                if (vpcId == null)
+                if (vpcId == null) {
                     throw new InvalidParameterValueException("Cannot specify CIDR when using network offering with external devices!");
+                }
             }
         }
 
         // Vlan is created in 1 cases - works in Advance zone only:
         // 1) GuestType is Shared
-        boolean createVlan = (startIP != null && endIP != null && zone.getNetworkType() == NetworkType.Advanced
-                && ((ntwkOff.getGuestType() == Network.GuestType.Shared)
-                || (ntwkOff.getGuestType() == GuestType.Isolated &&
-                !areServicesSupportedByNetworkOffering(ntwkOff.getId(), Service.SourceNat))));
+        boolean createVlan = (startIP != null && endIP != null && zone.getNetworkType() == NetworkType.Advanced && ((ntwkOff.getGuestType() == Network.GuestType.Shared)
```

Comment:
@rafaelweingartner What is the guideline of breaking lines? In many places line breaks were made when the length of the line was long, and here, for example, the final result makes the line length from short to long.

Reply:
That is a good question. I am using this code formatter here [1], which is available in our Wiki.
I did not take much attention on these details. I was expecting the formatter there to be the right one for us to use. The formatter is presented in this page here [2].

Would you like me to change some of the changes that were automatically applied by our formatter [1]?

[1] https://cwiki.apache.org/confluence/download/attachments/29687985/ApacheCloudStack.xml?version=1&modificationDate=1382960469000&api=v2
[2] https://cwiki.apache.org/confluence/display/CLOUDSTACK/Coding+conventions